### ISSUE TITLE
Add repetition penalty sampler

### ISSUE THREAD

**Issue by ybubnov** [ybubnov/metalchat#201]
Add a sampler that applies repetition penalty to the output logits.
Consider using `1.05` value in the LFM2.5 model inference.
